index.html

```diff
@@ -94,7 +94,8 @@ <h3 class="refs__links__title">Informace o klubu FIT++</h3>
         <a class="refs__links__link" href="/documents/volebni_rad_2020.pdf"><i class="refs__links__link__download ion ion-ios-cloud-download"></i> Volební řád FIT++</a>
 
         <h3 class="refs__links__title">Zápisy ze schůzí představenstva FIT++</h3>
-        <a class="refs__links__link" href="https://docs.google.com/document/d/1a_RNfiRt0GUxAqxVYZ5sdlkFAmSiFyBzajOUBJ42Bzg/edit"><i class="refs__links__link__download ion ion-ios-cloud-download"></i> 15. 5. 2023 - neschváleno</a>
+        <a class="refs__links__link" href="https://docs.google.com/document/d/1p2e_pi2nE2mKdW9AG_3NHuJieBxDKWfDGCGUxo8ZlEM/edit"><i class="refs__links__link__download ion ion-ios-cloud-download"></i> 29. 10. 2023 - neschváleno</a>
+		<a class="refs__links__link" href="/documents/zapis/230515.pdf"><i class="refs__links__link__download ion ion-ios-cloud-download"></i> 15. 5. 2023</a>
```

Comment:
Sus white space odsazení, máš tam 2 taby místo 8 mezer

Reply:
Jinak to je samozřejmě OK :ok_hand:

Reply:
mergenu a fiksnu

Reply:
pardon, notepad++ mam nastavenej na taby :rtzw:

Reply:
Všichni víme že taby jsou superior